trlx/model/accelerate_ilql_model.py

```diff
@@ -1,180 +1,179 @@
-import os
-from typing import Dict, Iterable
+from typing import Iterable, Union
```

Comment:
Unfortunately I really cannot review ilql so I trust things are fine here.

Perhaps it would be a good idea to write some unittests for the ilql implementation moving forward.